applications/nrf_desktop/doc/cpu_meas.rst

```diff
@@ -18,15 +18,15 @@ Module events
 Configuration
 *************
 
-Enable the module using ``CONFIG_DESKTOP_CPU_MEAS_ENABLE`` Kconfig option.
-This Kconfig option selects the :option:`CONFIG_CPU_LOAD`.
-:option:`CONFIG_CPU_LOAD` enables the :ref:`cpu_load` that is used to perform the measurements.
+Enable the module using the ``CONFIG_DESKTOP_CPU_MEAS_ENABLE`` Kconfig option.
+This Kconfig option selects the ``CONFIG_CPU_LOAD`` option.
```

Comment:
Changed to :option:`CONFIG_CPU_LOAD` since this option is not specific to nrf_desktop.